platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/generate-platform-diagram-with-inline-pces.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# The location were this script can be found.
+SCRIPT_PATH="$(dirname "$(readlink -f "$0")")"
+
+# You must install mermaid to use this script.
+# npm install -g @mermaid-js/mermaid-cli
+
+# Add the flag "--less-mystery" to add back labels for mystery input wires (noisy diagram warning)
+
+pcli diagram \
+    -l 'TransactionPrehandler:futures:TransactionHandler' \
+    -l 'EventCreationManager:get transactions:TransactionPool' \
+    -l 'ConsensusEventStream:future hash:TransactionHandler' \
+    -s 'EventWindowManager:event window:🌀' \
+    -s 'Heartbeat:heartbeat:❤️' \
+    -s 'TransactionPrehandler:futures:🔮' \
+    -s 'pcesReplayer:done streaming pces:✅' \
+    -s 'OrphanBufferSplitter:events to gossip:📬' \
```

Comment:
```suggestion
    -s 'InlinePcesWriter:events to gossip:📬' \
```

The old substitution isn't valid anymore